docs/contributing.md

```diff
@@ -1,35 +1,41 @@
 # Contributing
+
 All contributions are welcome!
 
 ## Bug Reporting
-If you find a bug, submit a bug report on GitHub Issues. 
+
+If you find a bug, submit a bug report on GitHub Issues.
 
 ## Adding Features/Fixing Bugs
+
 If you have identified a new feature or bug that you can fix yourself, please follow the following procedure.
 
-1. Clone `main` branch.
-2. Create a new branch to contain your changes. 
-2. `add`, `commit`, and `push` your changes to this branch. 
-3. Create a pull request (PR). See more information on submitting a PR request below.
+1. Fork `main` branch.
```

Comment:
Cloning does not work because outside people do not have write permission. One has to fork to contribute, which is standard practice for open-source repos.